_observing-your-data/alerting/per-document-monitors.md

```diff
@@ -0,0 +1,52 @@
+---
+layout: default
+title: Per document monitors
+nav_order: 20
+parent: Monitors
+grand_parent: Alerting
+has_children: false
+---
+
+# Per document monitors
+Introduced 2.0
+{: .label .label-purple }
+
+Per document monitors are a type of alert monitor that can be used to identify and alert on specific documents in an OpenSearch index. For example, you can use the monitor to:
+
+- Detect corrupted data or unauthorized changes.
+- Enforce data quality policies, such as ensuring all documents contain a certain field or that values in a field are within a certain range. 
+- Track changes to a specific document over time, which can be helpful for auditing and compliance purposes
+
+## Defining queries
+
+Per document monitors allow you to define up to 10 queries that compare a selected field with a desired value. You can define supported field data types using the following operators:
+
+- `is` 
+- `is not`
+- `is greater than`
+- `is greater than equal`
+- `is less than`
+- `is less than equal`
+
+You can query each [trigger]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/) using up to 10 tags, adding the tag as a single trigger condition instead of specifying a single query. The [Alerting plugin]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/monitors/) processes the trigger conditions from all queries as a logical `OR` operation, so if any of the query conditions are met, it triggers an alert. The Alerting plugin then tells the [Notifications plugin]({{site.url}}{{site.baseurl}}/observing-your-data/notifications/index/) to send the alert notification to a channel.
+
+You can only use _tags_, that is, labels that can be applied to multiple queries to combine them with the logical `OR`` operation, in a per document monitor.
```

Comment:
```suggestion
You can only use _tags_--- that is, labels that can be applied to multiple queries to combine them with the logical `OR`` operation---in a per document monitor.
```
This feels like a parenthetical phrase. If the formatting looks wonky, I'd just throw them in actual parentheses.